scripts/pr-label-reader.ps1

```diff
@@ -0,0 +1,39 @@
+param (
+    # The JSON string containing the labels to process - comes in this case from GitHub PR event
+    # and is filtered to only listen for PR closure by webhook in aks-cleanup-nonprod.yaml file.
+    [Parameter(Mandatory=$true)]
+    [string]$labels
+)
+
+$labelsJson = ConvertFrom-Json -InputObject $labels
+
+# Split the label to only collect value
+function Split-Label {
+    param([string]$label)
+    return $label.Split(":")[1]
+}
+
+# Iterate through PR labels to collect
+foreach ($label in $labelsJson) {
+    $labelName = $label.name
+    Write-Output "Label name: $labelName"
+
+    if ($labelName.StartsWith("ns:")) {
+        $namespace = Split-Label $labelName
+    }
+    if ($labelName.StartsWith("rel:")) {
+        $releaseName = Split-Label $labelName
+    }
+    if ($labelName.StartsWith("prd:")) {
```

Comment:
I think we definitely don't need this, but may be we are handling in the subsequent ticket anyway to handle this